src/scenes/Partners/Edit/EditPartner.tsx

```diff
@@ -128,6 +133,25 @@ const fieldMapping = {
   'partner.countries': ({ props }) => (
     <LocationField {...props} label="Countries" multiple variant="outlined" />
   ),
+  'partner.languageOfReportingId': ({ props }) => (
+    <LanguageField
+      {...props}
+      label="Language of Reporting"
+      CreateDialogForm={undefined}
+      getOptionDisabled={(option) => !option.isAvailableForReporting.value}
+    />
+  ),
+  'partner.languageOfWiderCommunicationId': ({ props }) => (
+    <LanguageField
+      {...props}
+      label="Language of Wider Communication"
+      CreateDialogForm={undefined}
+      getOptionDisabled={(option) => !option.isWiderComm.value}
+    />
+  ),
+  'partner.languagesOfConsulting': ({ props }) => (
+    <LanguageField {...props} multiple label="Languages of Consulting" />
```

Comment:
outlined variant looks better for multiple values aka chips. the gray chips on filled variant gray bg looks bad.
```suggestion
    <LanguageField {...props} multiple label="Languages of Consulting" variant="outlined" />
```